App.js

```diff
@@ -1,25 +1,77 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import styled from 'styled-components/native'
 
-const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
+import { Header } from './components/Header'
+import { Intro } from './components/Intro'
+import { CatList } from './components/CatList'
+import { Footer } from './components/Footer'
+
+const ScrollView = styled.ScrollView`
+  background-color: #fff5a5;
+`
+
+const Section = styled.View`
+  background-color: #fff5a5;
+  width: 100%;
+  display: flex;
   justify-content: center;
   align-items: center;
 `
 
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
+const Container = styled.View`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 80%;
 `
 
 const App = () => {
+
+  const [cats, setCats] = useState([])
+  const [isLoading, setLoading] = useState(true) //remove id I don't use a loading
```

Comment:
Super small issue, but nevertheless I think you should to write `setIsLoading` instead of `setLoading`

Reply:
updated